dynamiq/nodes/agents/base.py

```diff
@@ -107,6 +107,12 @@ class AgentIntermediateStep(BaseModel):
     final_answer: str | dict | None = None
 
 
+class ToolParams(BaseModel):
+    global_params: dict[str, Any] = Field(default_factory=dict, alias="global")
```

Comment:
maybe make sense to do this consisteny `global`, `by_name`, `by_id` or `global_params`, `by_name_params`, `by_id_params`
Or `global` reserved word/func in Pydantic?